examples/eco_force/sigma5.ori

```diff
@@ -0,0 +1,6 @@
+1.671685 0.557228 1.76212
+0.557228 -1.671685 1.76212
+2.228913 -1.114456 0.000000
+0.557228 1.671685 1.76212
+1.671685 -0.557228 1.76212
+2.228913 1.114456 0.000000
```

Comment:
please make sure that *all* files have a LF at the end of the file. files without result in short reads on some platforms.